src/portal-impl.c

```diff
@@ -587,19 +587,24 @@ find_portal_implementation (const char *interface)
         }
     }
 
-#if 0
-  /* Fall back to *any* installed implementation */
   for (l = implementations; l != NULL; l = l->next)
     {
       PortalImplementation *impl = l->data;
 
+      if (!g_str_equal (impl->dbus_name, "org.freedesktop.impl.portal.desktop.gtk"))
```

Comment:
It would be good to leave a comment, here, making it explicit that we consider the gtk portal to be the last resort, instead of "any installed implementation". I know `git blame` exists, but having a comment makes it more prominent.